dataplane/standalone/entrypoint.cc

```diff
@@ -118,21 +119,21 @@
 #include "dataplane/standalone/sai/virtual_router.h"
 #include "dataplane/standalone/sai/vlan.h"
 #include "dataplane/standalone/sai/wred.h"
-#include "dataplane/standalone/translator.h"
 
 extern "C" {
 #include "experimental/saiextensions.h"
 #include "inc/sai.h"
 }
 
-std::shared_ptr<Translator> translator;
 std::unique_ptr<lemming::dataplane::sai::Acl::Stub> acl;
 std::unique_ptr<lemming::dataplane::sai::Bfd::Stub> bfd;
 std::unique_ptr<lemming::dataplane::sai::Buffer::Stub> buffer;
 std::unique_ptr<lemming::dataplane::sai::Bmtor::Stub> bmtor;
 std::unique_ptr<lemming::dataplane::sai::Bridge::Stub> bridge;
 std::unique_ptr<lemming::dataplane::sai::Counter::Stub> counter;
 std::unique_ptr<lemming::dataplane::sai::DebugCounter::Stub> debug_counter;
+std::unique_ptr<lemming::dataplane::sai::GenericProgrammable::Stub>
+    generic_programmable;
```

Comment:
Where are these variables used?

Reply:
in the generated sai code. they are initialized here the grpc chan